docs/testrunner-toolkit/configuration/common-syntax.md

```diff
@@ -315,6 +316,16 @@ artifacts:
 ```
 ---
 
+### `cleanup`
+<p><small>| OPTIONAL | BOOLEAN |</small></p>
+
+Specifies whether to remove all contents of artifacts directory.
```

Comment:
How does this work? If set to true, all content (folders and files) of the specified download directory are cleared before any new artifacts from the current test are downloaded?

Reply:
Correct 👍

Reply:
yes. It will remove artifacts directory before starting this run